homeassistant/components/pooldose/config_flow.py

```diff
@@ -0,0 +1,154 @@
+"""Config flow for the Seko Pooldose integration."""
+
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+from pooldose.client import PooldoseClient
+from pooldose.request_handler import RequestHandler, RequestStatus
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.config_entries import ConfigFlow, ConfigFlowResult
+from homeassistant.const import CONF_HOST, CONF_SCAN_INTERVAL, CONF_TIMEOUT
+from homeassistant.helpers import config_validation as cv
+
+from .const import (
+    CONF_INCLUDE_SENSITIVE_DATA,
+    CONF_SERIALNUMBER,
+    DEFAULT_HOST,
+    DEFAULT_SCAN_INTERVAL,
+    DEFAULT_TIMEOUT,
+    DOMAIN,
+)
+
+_LOGGER = logging.getLogger(__name__)
+
+SCHEMA_DEVICE = vol.Schema(
+    {
+        vol.Required(CONF_HOST, default=DEFAULT_HOST): cv.string,
+        vol.Optional(
+            CONF_SCAN_INTERVAL, default=DEFAULT_SCAN_INTERVAL
+        ): cv.positive_int,
+        vol.Optional(CONF_TIMEOUT, default=DEFAULT_TIMEOUT): cv.positive_int,
+        vol.Optional(CONF_INCLUDE_SENSITIVE_DATA, default=False): cv.boolean,
```

Comment:
1. We should not have a configurable scan interval. Users should use https://www.home-assistant.io/common-tasks/general/#defining-a-custom-polling-interval to change their polling interval.
2. We should pick a timeout that works for 99% of people (preferably all)
3. Why is there an option to include sensitive data?

Reply:
yes. This way we can give users more flexibility, because then they can automate polling based on location or day/night or whatnot